### DISCUSSION TITLE
A doubt regarding Kubescape dashboard

### DISCUSSION THREAD

**Discussion by SD-13** [kubescape/kubescape#731] [General]
**I have a doubt. While visiting the Kubescape [dashboard](https://cloud.armosec.io/dashboard), for me it looks something like this.**

![Screenshot from 2022-08-29 23-41-29](https://user-images.githubusercontent.com/89520981/187270177-481876a3-1475-42fa-9224-17a6224d4ca6.png)

**I wanted to highlight this part**

![Screenshot from 2022-08-29 23-41-49](https://user-images.githubusercontent.com/89520981/187270190-168dfae4-0f7c-4aeb-a0f6-aaf2c9e0f08e.png)

**1. In 100% zoom, the bottom left part is missing some content
2. but if I zoom out it looks fine. I think it is not expected.
3. Can anyone please give me feedback so that I can raise an issue for this? Thanks,**

**System specification**
OS: Ubuntu 22.04 LTS
Browser: Chrome

**Comment by craigbox** [comment]
Thanks for the report!
Redesigns of the ARMO Platform have changed the display so this doesn't seem to be an issue any more.
